input/pagecontent/reference.md

```diff
@@ -1705,7 +1733,18 @@ Creating a code system uses the keywords `CodeSystem`, `Id`, `Title` and `Descri
       "Bhujangasana, or Cobra Pose is a reclining back-bending asana in hatha yoga and modern yoga as exercise. It is commonly performed in a cycle of asanas in Surya Namaskar (Salute to the Sun) as an alternative to Urdhva Mukha Svanasana (Upwards Dog Pose)."
   ```
 
-> **Note:** FSH does not support definition of relationships between local codes, such as parent-child (is-a) relationships.
+* Define a code system for anteater taxonomy.
+
+  ```
+  CodeSystem: AnteaterCS
+  Id: anteater-code-system
+  Title: "Anteater Code System."
+  * #Anteater "Anteater" "Members of suborder Vermilingua."
+  * #Anteater #Tamandua "Members of genus Tamandua"
+  * #Anteater #Tamandua #NorthernTamandua "Northern Tamandua"
+  * #Anteater #Tamandua #SouthernTamandua "Southern Tamandua"
+  * #Anteater #GiantAnteater "Giant Anteater"
```

Comment:
I think that, in order to represent best practice, we should define separate display and definition values for all of these.  Are you familiar enough with anteaters to be able to do that?

Reply:
On the AnteaterCS example, show how this will look with indented paths.

Reply:
`
  equivalent to (using [indented paths](#indented-paths)):

  ```
  CodeSystem: AnteaterCS
  Id: anteater-code-system
  Title: "Anteater Code System."
  * #Anteater "Anteater" "Members of suborder Vermilingua."
    * #Tamandua "Members of genus Tamandua"
      * #NorthernTamandua "Northern Tamandua"
      * #SouthernTamandua "Southern Tamandua"
    * #GiantAnteater "Giant Anteater"
  ```
`

Reply:
> I think that, in order to represent best practice, we should define separate display and definition values for all of these. Are you familiar enough with anteaters to be able to do that?

Absolutely not, so I'll have to do some research or choose a different topic for this example.

Reply:
Ha.  OK.  Here's my knowledge from wikipedia:
```
* #Anteater "Anteater" "Species of the suborder Vermilingua, distinguished by its propensity to eat ants"
  * #Tamandua "Tamandua" "The Tamandua genus of anteaters, mainly found in forests and grasslands"
    * #NorthernTamandua "Northern Tamandua" "The northern species of Tamandua anteaters"
    * #SouthernTamandua "Southern Tamandua" "The southern species of Tamandua anteaters"
  * #GiantAnteater "Giant Anteater" "The Giant Anteater, typically 6 - 7 feet in length"
```